src/Workload/Microsoft.Maui.Resizetizer.Sdk/Microsoft.Maui.Resizetizer.Sdk.csproj

```diff
@@ -1,9 +1,11 @@
+<!-- NOTE: Resizetizer.csproj is temporarily replacing usage of this file -->
```

Comment:
This file is not currently used in this PR anymore, but I left it here if we want to go back?